artdaq-mu2e/ArtModules/DTCEventVerifier_module.cc

```diff
@@ -102,13 +105,21 @@ bool mu2e::DTCEventVerifier::filter(art::Event& event)
     }
   
   
-  if (diagLevel_ > 10) 
+  if (diagLevel_ > 0) 
     {
-      std::cout << "[ArtFragmentsFromDTCEvents::produce] Found nHandlesnFragments  " 
+      std::cout << "[ArtFragmentsFromDTCEvents::produce] Found nFragments  " 
 		<< fragments.size() <<std::endl;
     }
-  
+  if (metricMan != nullptr)
+    {
+      std::ostringstream oss;
+      oss << "Found nFragments=  " << fragments.size();
+      metricMan->sendMetric("nFragments", oss.str(), "xml_string",
```

Comment:
Probably better to just do:
`metricMan->sendMetric("nFragments", fragments.size(), "Fragments", metrics_reporting_level_, artdaq::MetricMode::LastPoint);`
In general, numeric metrics should be preferred over string metrics where possible.

Reply:
thanks for the suggestion! I made the changes as you suggested.